src/commands/veloce/debug/push.ts

```diff
@@ -49,19 +73,17 @@ export default class Org extends DebugSfdxCommand {
     };
     const backendUrl: string | undefined = debugSession.backendUrl;
```

Comment:
You don't need to specify type, as `debugSession.backendUrl` is always string.
You can just remove this variable, and use `debugSession.backendUrl` below.